src/dbnode/storage/bootstrap/bootstrapper/commitlog/source.go

```diff
@@ -261,9 +301,7 @@ func (s *commitLogSource) Read(
 			shardTimeRanges := ns.DataRunOptions.ShardTimeRanges
 			dataResult = shardTimeRanges.ToUnfulfilledDataResult()
 		}
-		var indexResult result.IndexBootstrapResult
```

Comment:
Now each `bootstrap.NamespaceResult` will have the same `IndexResult`. Is that ok?

Reply:
Ah, good catch the index results should be per ns.